pkgs/native_toolchain_c/lib/src/cbuilder/cbuilder.dart

```diff
@@ -57,18 +57,27 @@ class CBuilder implements Builder {
   @visibleForTesting
   final Uri? installName;
 
+  /// Whether the compiler will emit position independent code.
+  ///
+  /// When set to `null`, the default behavior of the compiler will be used.
+  ///
+  /// Defaults to `true` for libraries and `false` for executables.
+  final bool? pic;
```

Comment:
Add a comment that this means PIE for executables

Reply:
Done